Source/WebCore/rendering/RenderElement.h

```diff
@@ -304,7 +304,7 @@ class RenderElement : public RenderObject {
     void clearNeedsLayoutForDescendants();
 
 protected:
-    enum BaseTypeFlag {
+    enum class RenderElementType {
```

Comment:
This named this RenderElementType instead of Type because I want to move it to RenderObject in a follow-up.